tests/foreman/cli/test_contentview.py

```diff
@@ -4688,14 +4702,16 @@ def make_file_repository_upload_contents(self, module_org, module_product, optio
         if options is None:
             options = {'product-id': module_product.id, 'content-type': 'file'}
         if not options.get('content-type'):
-            raise CLIFactoryError('Please provide a valid Content Type.')
-        new_repo = make_repository(options)
-        remote_path = f"/tmp/{RPM_TO_UPLOAD}"
+            raise cli_factory.CLIFactoryError('Please provide a valid Content Type.')
+        new_repo = cli_factory.make_repository(options)
+        remote_path = f"/tmp/{constants.RPM_TO_UPLOAD}"
```

Comment:
```suggestion
        remote_path = f'/tmp/{constants.RPM_TO_UPLOAD}'
```

Reply:
Cleaned up some other uses of format in this module.